release_notes/ocp_3_3_release_notes.adoc

```diff
@@ -1343,6 +1343,43 @@ To upgrade an existing {product-title} 3.2 or 3.3 cluster to this latest release
 automated upgrade playbook. See
 xref:../install_config/upgrading/automated_upgrades.adoc#running-the-upgrade-playbook-directly[Performing Automated In-place Cluster Upgrades] for instructions.
 
+[[ocp-3-3-1-3-technology-preview]]
+==== Technology Preview Features
+
+Scheduled Jobs::
+Scheduled jobs build upon the xref:../dev_guide/jobs.html#dev-guide-jobs[job]
+object by allowing you to specifically schedule how the job should be run. See
+xref:../dev_guide/scheduled_jobs.adoc#dev-guide-scheduled-jobs[Scheduled Jobs]
+for more details.
+
+Sysctl Support::
+Sysctl settings are now exposed via Kubernetes, allowing users to modify kernel
+parameters at runtime for namespaces within a container. This can be done by
+setting annotations on pods, for example:
+
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: sysctl-example
+  annotations:
+    security.alpha.kubernetes.io/sysctls: kernel.shm_rmid_forced=1
+----
+
+Sysctls that are namespaced are considered _safe_ and supported. Sysctls that
```

Comment:
"Sysctls that are namespaced are considered safe and supported." is wrong. Only a small subset of namespaced sysctls are safe to be used in a container environment, namely those which cannot be misused to influence other containers, e.g. by blocking resources like memory outside of the pods' defined memory limits. A rule of thumb: if an exposed sysctl is a security threat in a multi-tenant environment, it is not considered safe.